dom0-updates/qubes-dom0-updates.cron

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+qvm-features dom0 last-updates-check "$(date +'%Y-%m-%d %T')"
```

Comment:
Move this after the chunk below? Otherwise it will report updates were checked when in fact it was disabled.